packages/insomnia/src/ui/css/components/request-url-bar.less

```
@@ -58,6 +54,10 @@
   & > .dropdown {
     height: 100%;
   }
+  .urlbar__send-context,
+  .surprise {
+    border-left: 1px solid var(--hl);
```

Should we use the same color as the other borders in the app?
```suggestion
border-left: 1px solid var(--hl-md);
```